xoinvader/animation.py

```diff
@@ -0,0 +1,130 @@
+"""Animation.
+
+Animation is set of keyframes.
+Value of selected attribute changes in time.
+
+Keyframe:
+  (time, value)
+
+Objects have animation manager which manages animation graph and switching."""
+
+
+from operator import itemgetter
+
+from xoinvader.utils import Timer
+
+
+# TODO: Implement animation graph and etc
+class AnimationManager(object):
+    """Manage list of object animation."""
+
+    def __init__(self):
+        self._animations = {}
+        self._animation = None
+
+    @property
+    def animation(self):
+        """AnimationManager's current animation.
+
+        :getter: yes
+        :type getter: str
+        :setter: yes
+        :type setter: :class:`xoinvader.animation.Animation`
+        """
+
+        if self._animation:
+            return self._animation.name
+        else:
+            raise AttributeError("There is no available animation.")
+
+    @animation.setter
+    def animation(self, name):
+        if name in self._animations:
+            self._animation = self._animations[name]
+        else:
+            raise ValueError("No such animation: '{0}'.".format(name))
+
+    def add(self, name, *args, **kwargs):
+        """Add new animation, pass args to Animation class.
+
+        :param str name: animation name
+        """
+
+        animation = Animation(name, *args, **kwargs)
+        self._animations[name] = animation
+
+        if len(self._animations) == 1:
```

Comment:
I think way of this check doesn't matter in both cases.

* we removed all animations
* adding new first animation-> use it because we have single animation